app/src/androidTestWebview/java/org/mozilla/focus/privacy/WebViewDataTest.java

```diff
@@ -185,30 +185,12 @@ public void DeleteWebViewDataTest() throws InterruptedException, UiObjectNotFoun
         Assert.assertTrue(TestHelper.webView.waitForExists(waitingTime));
 
         // Assert website is loaded
-        /*
-        final UiObject titleMsg = TestHelper.mDevice.findObject(new UiSelector()
-                .description("focus test page")
-                .enabled(true));
-        titleMsg.waitForExists(waitingTime);
-        assertTrue("Website title loaded", titleMsg.exists());
-
-        // Assert cookie is saved
-
-        final UiObject cookieMsg = TestHelper.mDevice.findObject(new UiSelector()
-                .description("Cookie saved")
-                .enabled(true));
-        cookieMsg.waitForExists(waitingTime);
-        assertTrue("Cookie is saved", cookieMsg.exists());
-        final UiObject serviceWorkerMsg = TestHelper.mDevice.findObject(new UiSelector()
-                .description("Service worker installed")
-                .enabled(true));
-        serviceWorkerMsg.waitForExists(waitingTime);
-        assertTrue("Service worker installed", serviceWorkerMsg.exists());
-*/
+        // check for disappearance of progress bar (as content won't load in geckoview)
+        TestHelper.progressBar.waitUntilGone(waitingTime);
+
         // Erase browsing session
         TestHelper.floatingEraseButton.perform(click());
         TestHelper.erasedMsg.waitForExists(waitingTime);
-        Assert.assertTrue(TestHelper.erasedMsg.exists());
```

Comment:
removing assert here as the message disappears too quickly to make a reliable assert